open-metadata-implementation/access-services/digital-service/digital-service-server/src/main/java/org/odpi/openmetadata/accessservices/digitalservice/server/DigitalServiceRESTServices.java

```diff
@@ -0,0 +1,79 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+/* Copyright Contributors to the ODPi Egeria project. */
+package org.odpi.openmetadata.accessservices.digitalservice.server;
+
+
+import org.odpi.openmetadata.accessservices.digitalservice.handlers.DigitalServiceEntityHandler;
+import org.odpi.openmetadata.accessservices.digitalservice.properties.DigitalService;
+import org.odpi.openmetadata.accessservices.digitalservice.rest.DigitalServiceRequestBody;
+import org.odpi.openmetadata.commonservices.ffdc.RESTExceptionHandler;
+import org.odpi.openmetadata.commonservices.ffdc.rest.GUIDResponse;
+import org.odpi.openmetadata.frameworks.connectors.ffdc.InvalidParameterException;
+import org.odpi.openmetadata.frameworks.connectors.ffdc.PropertyServerException;
+import org.odpi.openmetadata.frameworks.connectors.ffdc.UserNotAuthorizedException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+
+
+/**
+ * The DigitalServiceRESTServices provides the server-side implementation of the Stewardship Action Open Metadata
+ * Assess Service (OMAS).  This interface provides connections to assets and APIs for adding feedback
+ * on the asset.
+ */
+public class DigitalServiceRESTServices
+{
+    private static DigitalServiceInstanceHandler instanceHandler = new DigitalServiceInstanceHandler();
+
+    private static final Logger log = LoggerFactory.getLogger(DigitalServiceRESTServices.class);
+
+    private RESTExceptionHandler restExceptionHandler = new RESTExceptionHandler();
+
+    /**
+     * Default constructor
+     */
+    public DigitalServiceRESTServices()
+    {
+    }
+
+
+    /**
+     * Create deployed database schema guid response.
+     *
+     * @param serverName                        the server name
+     * @param userId                            the user id
+     * @param digitalServiceRequestBody the deployed database schema request body
+     * @return the guid response
+     */
+    public GUIDResponse createDigitalService( String userId,
+                                              String serverName,
+                                              DigitalServiceRequestBody digitalServiceRequestBody) {
+
+        final String methodName = "createDigitalService";
```

Comment:
Missing logging 
```
        final String   methodName = "createDigitalService";

        RESTCallToken token = restCallLogger.logRESTCall(serverName, userId, methodName);
          .
          .
       restCallLogger.logRESTCallReturn(token, response.toString());
        return response;
```

Not catching `Throwable`.  Need to ensure no exception stacks leave the server.

Reply:
resolved

Reply:
added